docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -180,22 +181,17 @@ hyperparameter optimization to give minimum validation errors.
 ===== Standard parameters
 
 `dependent_variable`::
```

Comment:
Don't we want to reuse the definition from ml-shared?

Reply:
@dimitris-athanasiou The definition of the `dependent_variable` is not identical for regression (only numeric field) and for classification (numeric or boolean field), that's why I eventually didn't use the definition from ml-shared. A possible workaround came to my mind, my last commit added it.